raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -22,6 +22,8 @@ contract TokenNetwork is Utils {
     // Chain ID as specified by EIP155 used in balance proof signatures to avoid replay attacks
     uint256 public chain_id;
 
+    uint256 constant public MAX_SAFE_UINT256 = 115792089237316195423570985008687907853269984665640564039457584007913129639935;
```

Comment:
I was reluctant to use `2 ** 256 - 1`, even though it works correctly now, because `2 ** 256` is not a valid value for `uint256`. 
I was reluctant to use `0 - 1` in case anything changes in the EVM regarding how underflows are handled.

I thought that having the actual number there is on the safer side. Maybe an overkill to think about future EVM changes, but the current implementation should be fine.

Reply:
I asked because `115792089237316195423570985008687907853269984665640564039457584007913129639935` is a number that I don't know by heart, so I had to run `2 ** 256 - 1` to check the value was correct.

but I can see your point, either way is fine to me

Reply:
I also added a test comparing this number with `2 ** 256 - 1` (python) and with `0 - 1` (Solidity)

Reply:
> I was reluctant to use 0 - 1 in case anything changes in the EVM regarding how underflows are handled.

Stuff like that won't change anytime soon. It would break way too many production systems.